File: Atividades c#/atividade4.cs
using System;
class ativ4
{
    static void Main()
    {
        int a, b, c;
        Console.WriteLine("digite dois valores:");
        a = int.Parse(Console.ReadLine());
        b = int.Parse(Console.ReadLine());
        c = a + b;
        Console.WriteLine($"a soma de {a} + {b} = {c}");

    }
}

File: Atividades Java/atividade4.java
import java.util.Scanner;

public class atividade4 
{

    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        int valor1,valor2,result;

        System.out.println("Digite dois numeros:");
        valor1 = scan.nextInt();
        valor2 = scan.nextInt();

        result = valor1 + valor2;

        System.out.println("A soma dos valores digitados: " + result);
        
    }

}
